integ/components/deadline/common/scripts/bash/deploy-utils.sh

```diff
@@ -20,9 +20,13 @@ function deploy_component_stacks () {
   echo "Running $COMPONENT_NAME end-to-end test..."
 
   echo "Deploying test app for $COMPONENT_NAME test suite"
-  npx cdk deploy "*" --require-approval=never
+  if [ "${RUN_TESTS_IN_PARALLEL}" = true ]; then
+    npx cdk deploy "*" --require-approval=never > "$INTEG_TEMP_DIR/${COMPONENT_NAME}_deploy.txt" 2>&1
```

Comment:
I think it would be valuable to have an option to specify another directory to store these log files instead of the integ temp directory. Maybe having an optional environment variable like `PARALLEL_RESULTS_DIRECTORY`, with the default being the integ temp directory.

This makes it so these files don't get cleaned up by the `cleanup.sh` script that gets run after the tests, making it easier to look at the logs for each test rather than having one large file (or lots of output on the command line). What do you think?

Reply:
As I know the only file that appear in this folder is output from test in json format.
I can't see any reason why others files should be put in different folder. As for me they should be beside this output file.

Reply:
My idea was to have an option to keep these new `.txt` files as artifacts after a run of the integration tests to make it easier to inspect the results after the test. I did this myself when running the tests in parallel and thought it would be nice to have as an option for our tests.

This is more of a QOL option and isn't strictly necessary, so I won't hold up the PR for this.